modules/n1ql/pages/n1ql-language-reference/alterindex.adoc

```diff
@@ -386,10 +387,9 @@ include::ROOT:partial$query-context.adoc[tag=section]
 
 .Move the `def_inventory_airport_faa` index from one node to another
 ====
-Create a cluster of 3 nodes and then go to menu:Settings[Sample buckets] to install the `travel-sample` bucket.
+Create a cluster of 3 nodes and then go to menu:Settings[Sample buckets] to install the `<bucket>` bucket.
```

Comment:
Is this actually `<bucket>` for the user? Is there a better value we can use to make it clear that it changes based on the user's setup?